client/components/promo-section/index.tsx

```diff
@@ -12,6 +12,7 @@ import PromoCardCta, { Props as PromoCardCtaProps } from './promo-card/cta';
 interface PromoSectionCardProps extends PromoCardProps {
 	body: string;
 	cta?: PromoCardCtaProps;
+	learnMoreLink?: PromoCardCtaProps;
```

Comment:
Is there a mistake here?

```suggestion
	learnMoreLink: PromoCardCtaProps['learnMoreLink'];
```

Reply:
You're right the types got a bit confused here. I've updated them, although I'm not sure if there is a better property name to use than `ctaProps`. I'd be interested to know if you think this is better.